test/integration/redis_proxy_integration_test.cc

```diff
@@ -0,0 +1,83 @@
+#include "test/integration/redis_proxy_integration_test.h"
+
+#include <sstream>
+#include <vector>
+
+#include "extensions/filters/network/redis_proxy/command_splitter_impl.h"
+
+namespace RedisCommon = Envoy::Extensions::NetworkFilters::Common::Redis;
+namespace RedisCmdSplitter = Envoy::Extensions::NetworkFilters::RedisProxy::CommandSplitter;
+
+namespace Envoy {
+namespace {
+
+std::string makeBulkStringArray(std::vector<std::string>&& command_strings) {
+  std::stringstream result;
+
+  result << "*" << command_strings.size() << "\r\n";
+  for (uint64_t i = 0; i < command_strings.size(); i++) {
+    result << "$" << command_strings[i].size() << "\r\n";
+    result << command_strings[i] << "\r\n";
+  }
+
+  return result.str();
+}
+
+INSTANTIATE_TEST_SUITE_P(IpVersions, RedisProxyIntegrationTest,
+                         testing::ValuesIn(TestEnvironment::getIpVersionsForTest()),
+                         TestUtility::ipTestParamsToString);
+
+void RedisProxyIntegrationTest::initialize() {
+  config_helper_.renameListener("redis_proxy");
+  BaseIntegrationTest::initialize();
+}
+
+TEST_P(RedisProxyIntegrationTest, SimpleRequestAndResponse) {
```

Comment:
nit: please add a `//` comment above each test with a brief description of what the test does. We do this in all new tests.

Reply:
done